examples/broadcast.html

```diff
@@ -100,7 +100,33 @@
       console.log(err, result);
     }
   );
+
+  /** Resteem a post */ 
+  var username = 'guest123'; // Your account
+  /* For resteeming it must(!) be your posting key, otherwise it will fail */
+  var postingWif = '5JRaypasxMx1L97ZUX7YuC5Psb5EAbF821kkAGtBj7xCJFQcbLg'; // Your posting key
+  var account = 'dan'; // Author of post
+  var link ='https://steemit.com/steem/@dan/steemit-s-evil-plan-for-cryptocurrency-world-domination' // Post to resteem  
```

Comment:
Can you resteem your own posts?

Reply:
Probably not. It might work programmatically but the option is not available on the frontend.